pulumitest/pulumiTest.go

```diff
@@ -71,11 +72,19 @@ func (a *PulumiTest) Source() string {
 	return a.source
 }
 
-// T returns the current testing.T instance.
-func (a *PulumiTest) T() *testing.T {
+// PT returns the current PT instance.
+func (a *PulumiTest) PT() PT {
```

Comment:
Why rename the method too?

Given that this is kinda a breaking change, I think we should just remove this public accessor and use the `t` private field ourselves. I don't think it actually needs to be exposed.

Reply:
This is 100% not a breaking change anymore, not sure which program is going to break. The renamed method guarantees that anything out there calling `T()` is not affected. All existing programs pass *testing.T anyway. 

I wish we had an easy way to scan across our repos to do see if something's used or not, I suspect it's not used.

Reply:
> just remove this public accessor

This doesn't work across Go packages apparently.

Reply:
I don't think we need it available across packages

Reply:
See https://github.com/pulumi/providertest/pull/80 for an example of this change.